lib/mission_control/jobs/engine.rb

```diff
@@ -103,10 +103,12 @@ class Engine < ::Rails::Engine
 
       initializer "mission_control-jobs.importmap", after: "importmap" do |app|
         MissionControl::Jobs.importmap.draw(root.join("config/importmap.rb"))
-        MissionControl::Jobs.importmap.cache_sweeper(watches: root.join("app/javascript"))
+        if app.config.reloading_enabled?
```

Comment:
Oh, great catch! This looks correct to me 👍 I see that in [the original `importmap-rails` gem](https://github.com/rails/importmap-rails/blob/d91d5e134d3f27e2332a8cb2ac015ea03d130621/lib/importmap/engine.rb#L33) they use this condition instead:
```ruby
if app.config.importmap.sweep_cache && !app.config.cache_classes
```
Maybe this one is better? At least for consistency 🤔

Reply:
```ruby
      def reloading_enabled?
        enable_reloading
      end

      def enable_reloading
        !cache_classes
      end
```

So `reloading_enabled?` is  `!cache_classes` but with a more readable name, so I think if anything we might as well change it on the `import-rails` side.

I'll add the `sweep_cache` condition though.